message/messaging_service.cc

```diff
@@ -1301,8 +1295,17 @@ void messaging_service::register_node_ops_cmd(std::function<future<node_ops_cmd_
 future<> messaging_service::unregister_node_ops_cmd() {
     return unregister_handler(messaging_verb::NODE_OPS_CMD);
 }
-future<node_ops_cmd_response> messaging_service::send_node_ops_cmd(msg_addr id, node_ops_cmd_request req) {
+future<node_ops_cmd_response> messaging_service::send_node_ops_cmd(netw::msg_addr id, node_ops_cmd_request req) {
     return send_message<future<node_ops_cmd_response>>(this, messaging_verb::NODE_OPS_CMD, std::move(id), std::move(req));
 }
 
 } // namespace net
+
+namespace std {
```

Comment:
why shall we put the operator into `std` namespace? IMHO, we just need to put it into `new` namespace. as ADL looks up the operator defined in the argument's namespace.

Reply:
you mean `netw` namespace, right?
alright with me, I wasn't aware of that.

Reply:
sorry, typo. yeah, i meant `netw`. the first parameter is in the `std` namespace, that's why ADL works also, but i think semantically, we intend to define an operator for `netw::msg_addr` not for the first parameter. hence this suggestion. see also https://en.cppreference.com/w/cpp/language/adl